CHANGELOG.md

```diff
@@ -37,11 +44,6 @@ those writing mocha tests of Apostrophe modules.
 ### Fixes
 * Fix child page slug when title is deleted
 
-### Fixes
-
-* Fix various issues on conditional fields that were occurring when adding new widgets with default values or selecting a falsy value in a field that has a conditional field relying on it.  
-Populate new or existing doc instances with default values and add an empty `null` choice to select fields that do not have a default value (required or not) and to the ones configured with dynamic choices.
-
```

Comment:
this block refers to something that is not released yet... bad merge

Reply:
Agreed

Reply:
Not sure how it's happening when that work has already been released... please review what happened there and remerge current main and reopen

Reply:
@boutell this PR branch already contains the latest version of `main`.
I have just moved this block from the released section to the unreleased one